samza-kafka/src/test/scala/org/apache/samza/checkpoint/kafka/TestKafkaCheckpointManager.scala

```diff
@@ -19,22 +19,21 @@
 
 package org.apache.samza.checkpoint.kafka
 
-import kafka.admin.AdminUtils
-import kafka.common.{InvalidMessageSizeException, UnknownTopicOrPartitionException}
-import kafka.message.InvalidMessageException
-import kafka.server.{KafkaConfig, KafkaServer, ConfigType}
-import kafka.utils.{CoreUtils, TestUtils, ZkUtils}
-import kafka.integration.KafkaServerTestHarness
-
-import org.apache.kafka.common.security.JaasUtils
+import _root_.kafka.admin.AdminUtils
```

Comment:
Why's this `_root_` convention needed?

Reply:
I am not even sure what this _root_ is needed for. This is some Intelij's "import optimize" magic. 
Removed.